draft-dekater-scion-pki.md

```diff
@@ -1303,8 +1335,42 @@ The steps required to create a new AS certificate are the following:
 
 # Security Considerations
 
-The entire document is about security considerations.
-More details will follow in future versions of this draft.
+This section describes the possible security risks and attacks that SCION's control-plane PKI may be prone to, and how these may be mitigated. As SCION is an inter-domain network architecture, the focus lies on *inter*-AS trust issues. All *intra*-AS trust- and security aspects are out of scope.
+
+**Note:** This section only discusses security considerations related to SCION's control-plane PKI. For SCION control plane- and routing-specific security considerations, see {{I-D.scion-cp}}. {{I-D.scion-dp}} includes security considerations that concern the SCION data plane and data forwarding.
+
+
+## Kill Switches
+
+SCION’s trust architecture fundamentally differs from a global monopolistic trust model. In SCION, each ISD manages its own trust roots instead of a single global entity providing those roots. This structure gives each ISD autonomy in terms of key management and in terms of trust. This prevents SCION from the occurrence of a global kill switch affecting all ISDs at once. However, local kill switches are to some extent still possible in SCION. The following sections explain these cases and possible countermeasures.
+
+
+### Local ISD Kill Switch
+
+Executing a kill switch inside a local ISD can be done at different levels of the AS-level hierarchy. But compared to DNSSEC and BGPsec, a parent authority cannot switch off SCION core ASes, as they manage their own cryptographic trust roots. Another difference with DNSSEC and BGPsec is that the attack vector of intra-ISD kill switches has only two entry levels: All ASes within an ISD obtain certificates directly from the CAs included in the ISD's TRC. If one of the core’s root keys is compromised, an adversary could issue illegitimate AS certificates, which may be used in further attacks. Maliciously change the TRC through a TRC update is more complicated, because each TRC update requires multiple different voting keys (defined by the voting quorum in the TRC).
```

Comment:
I think this paragraph is the main part of this security considerations section. IMO it would make sense to expand this, discussing the results of compromised keys for every level of the CP-PKI.

Reply:
@matzf: As already mentioned a couple of comments above, I have tried to implement your suggestion about discussing it on each CP-PKI level. As I am not sure if I got it right, it would be great if you could check this. Thanks!